week3/week3-homework/Program.cs

```diff
@@ -0,0 +1,142 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+
+
+     var celsiusTemperature = new Temperature(17.00m);
+
+        string MakeSound (IAnimal animal){
+          return $"{animal.Name} says {animal.Sound}";
+        }  
+
+        var Cat = new Animal("Cat","meow meow");

```

Comment:
These are objects (instances of the Animal class). The assignment was to create classes that implements the IAnimal interface. Like 
public class Cat : IAnimal { .... }